test/models/Author.spec.js

```diff
@@ -0,0 +1,85 @@
+import assert from 'power-assert';
+import mockgoose from 'mockgoose';
+import Promise from 'bluebird';
+import AutherModel from 'models/Author';
+
+describe('/models/Author', ()=> {
+  afterEach(()=> {
+    mockgoose.reset();
+  });
+
+  it('Authorコレクションは自動で付与されたpageIDを持つ', (done)=> {
+    const author = new AutherModel({
+      name: '和月伸宏',
+      lastModified: new Date(),
+    });
+    author.save((saveError)=> {
```

Comment:
__[jshint]__
*  'arrow function syntax (=>)' is only available in ES6 (use esnext option).